canonical-web-and-design/vanilla-framework
# Upgrade guide

## Done

- Cleaned up the upgrade guide to make it more current, in preparation for the release
- Added a link to the upgrade guide to the docs side nav

Fixes #1024

## QA

- Open [demo](/docs)
- Find the link to the upgrade guide in the side nav, click it
- See that the docs make sense to someone looking to upgrade

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
